app/app/explore/components/TransactionConfirmationModal.tsx

```diff
@@ -4,20 +4,39 @@ import {
   PositionStats,
   PositionStatus,
 } from "@/hooks/dataTransformers/positionsTransformer";
+import { useMemo } from "react";
 
 interface TransactionConfirmationModalProps {
   position: PositionStats | null;
-  txHash: string;
+  assetHash: string | null;
+  runeHash?: string | null;
```

Comment:
The type should be `runeHash: string | null` ? or can the value be not only `null` but also `undefined` ?

Reply:
Can be also undefined to keep `runeHash` optional.